### ISSUE TITLE
[FIX] User specific code editing

### ISSUE THREAD

**Issue by gazoscalvertos** [gazoscalvertos/Hass-Custom-Alarm#23]

**Comment by Crookedconverse** [comment]
I still am unable to add an additional user. Maybe I'm not doing it how it's suppose to be done. I did notice the [on-click=''] didn't have any data inside of it.  Is this intended? I thought I might have to define users somewhere else but I'm just not sure where I should define users or how to define them.							
```
783    <h1>Users Defined Codes</h1>
784    <paper-icon-button icon='mdi:account-settings-variant' data-type='button' data-setting='' on-click=''></paper-icon-button>
785    <paper-icon-button icon='mdi:account-plus' data-type='button' data-setting='' on-click=''></paper-icon-button>
786    <template is='dom-repeat' items='[[alarm.attributes.users]]' as='user'>
```

**Comment by gazoscalvertos** [comment]
Hi,

The multi user functionality is yet to be implemented in the settings page. This is next on the list to crack.

I'll keep you posted.

Thanks

**Comment by gazoscalvertos** [comment]
Implemented in 1.3.2
